app/viewmodels/SchemeDetailsTaskList.scala

```diff
@@ -45,7 +45,7 @@ object SchemeDetailsTaskList {
   implicit val formats: OFormat[SchemeDetailsTaskList] = Json.format[SchemeDetailsTaskList]
 }
 
-case class SchemeDetailsTaskListSection(isCompleted: Option[Boolean] = None, link: Link, header: Option[String] = None)
+case class SchemeDetailsTaskListSection(isCompleted: Option[Boolean] = None, link: Link, header: Option[String] = None, paragraphOne: Option[String] = None)
```

Comment:
Just a suggestion, but it would be more consistent with other code to call it p1 rather than paragraphOne. See h1/h2/h3 in SchemeDetailsTaskList.

Reply:
Changed